ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -59,15 +59,20 @@ images:
     ci_tests-private-baseui_4.18:
       as:
       - registry.ci.openshift.org/ci/tests-private-baseui:4.18
+    netobserv_network-observability-cli_ci:
```

Comment:
This needs to be in test-private-newpr image stanza

Reply:
oh yeah my bad